components/scripts/lib/build-scan-parse.sh

```diff
@@ -43,35 +43,39 @@ parse_build_scan_csv() {
 
   # shellcheck disable=SC2034 # not all scripts use all of the fetched data
   while IFS=, read -r field_1 field_2 field_3 field_4 field_5 field_6 field_7 field_8 field_9 field_10 field_11 field_12 field_13 field_14 field_15 field_16 field_17 field_18 field_19; do
-     if [[ "$header_row_read" == "false" ]]; then
-         header_row_read=true
-         continue;
-     fi
+    if [[ "$header_row_read" == "false" ]]; then
+       header_row_read=true
+       continue;
+    fi
 
-  if [[ "$build_cache_metrics_only" != "build_cache_metrics_only" ]]; then
-       project_names[idx]="$field_1"
-       base_urls[idx]="$field_2"
-       build_scan_urls[idx]="$field_3"
-       build_scan_ids[idx]="$field_4"
-       git_repos[idx]="$field_5"
-       git_branches[idx]="$field_6"
-       git_commit_ids[idx]="$field_7"
-       requested_tasks[idx]="$(remove_clean_task "${field_8}")"
-       build_outcomes[idx]="$field_9"
-       remote_build_cache_urls[idx]="${field_10}"
-       remote_build_cache_shards[idx]="${field_11}"
-     fi
+    if [[ "$build_scan_publishing_mode" == "off" ]]; then
+      project_names[idx]="$field_1"
+    fi
```

Comment:
This is the only meaningful change to this file. All other changes in this file are fixes to indentation which was all over the place.